README.md

```diff
@@ -385,6 +385,54 @@ command.
 Other options like `--wait-completion` are also available and work in the same way as for the `cloudos job run` command.
 Check `cloudos bash job --help` for more details.
 
+#### Get path to logs of job from CloudOS
+
+Get the path where Nextflow logs, Nextflow standard output, and trace files. It can be used on jobs with any status.
+
+Example
+```console
+cloudos job logs --cloudos-url $CLOUDOS \
+    --apikey $MY_API_KEY \
+    --workspace-id $WORKSPACE_ID \
+    --job-id "12345678910"
+    
+CloudOS python package: a package for interacting with CloudOS.
+
+Version: 2.25.0
+
+CloudOS job functionality: run, check and abort jobs in CloudOS.
+
+Executing logs...
+Logs URI: s3://path/to/location/of/logs
+
+Nextflow log: s3://path/to/location/of/logs/.nextflow.log
+
+Nextflow standard output: s3://path/to/location/of/logs/stdout.txt
+
+Trace file: s3://path/to/location/of/logs/trace.txt
+```
+
+#### Get path to result files of jobs from CloudOS
+
+Get the path where CloudOS stores the output files for a job. This can be used only for jobs with "completed" status
```

Comment:
```suggestion
Get the path where CloudOS stores the output files for a job. This can be used only on your user's jobs and for jobs with "completed" status.
```